sys/src/main/java/com/alibaba/nacos/sys/env/EnvUtil.java

```diff
@@ -99,15 +104,36 @@ public class EnvUtil {
     private static final String NACOS_TEMP_DIR_1 = "data";
     
     private static final String NACOS_TEMP_DIR_2 = "tmp";
-    
+
+    private static final String NACOS_CUSTOM_ENVIRONMENT_ENABLED = "nacos.custom.environment.enabled";
+
+    private static final String NACOS_CUSTOM_CONFIG_NAME = "customFirstNacosConfig";
+
     @JustForTest
     private static String confPath = "";
     
     @JustForTest
     private static String nacosHomePath = null;
     
     private static ConfigurableEnvironment environment;
-    
+
+    /**
+     * customEnvironment.
+     */
+    public static void customEnvironment() {
+        Boolean enableCustom = getProperty(NACOS_CUSTOM_ENVIRONMENT_ENABLED, Boolean.class);
+        if (Boolean.TRUE.equals(enableCustom)) {
```

Comment:
`if (enableCustom) ` is enough?

Reply:
Yes, that's enough. I'll fix it

Reply:
```
boolean enableCustom = getProperty(NACOS_CUSTOM_ENVIRONMENT_ENABLED, Boolean.class, false);
if (enableCustom)
```
I remember EnvUtil has method with Classes and default value.

Reply:
Yes, there is, but I don't think there is much difference between the two methods.